templates/logsearch-deployment.yml

```diff
@@ -0,0 +1,61 @@
+---
+meta:
+  <<: (( merge ))
+  environment: ~
+  default_env:
+    # Default vcap & root password on deployed VMs (ie c1oudc0w)
+    # Generated using mkpasswd -m sha-512
+    bosh:
+      password: (( merge || "$6$4gDD3aV0rdqlrKC$2axHCxGKIObs6tAmMTqYCspcdvQXh3JJcvWOY2WGb4SrdXtnCyNaWlrf3WEqvYR2MYizEGp3kMmbpwBC6jsHt0" ))
+  stemcell: (( merge ))
+
+name: (( meta.environment ))
+director_uuid: (( merge ))
+properties: (( merge ))
+
+jobs: (( merge ))
+networks: (( merge ))
+
+releases:
+- name: logsearch
+  version: latest
+
+compilation:
+  workers: 4
+  network: default
+  reuse_compilation_vms: true
+  cloud_properties: ~
+
+update:
+  serial: true
+  canaries: 1
+  canary_watch_time: 30000-600000
+  update_watch_time: 5000-600000
+  max_in_flight: 1
+  max_errors: 1
+
+resource_pools: 
+- name: logsearch
+  network: default
+  stemcell: (( meta.stemcell ))
+  cloud_properties: (( merge ))
+  env: (( merge || meta.default_env ))
+
+- name: logsearch_api
```

Comment:
Out of curiosity, is there background or a reason for the resource pools all being prefixed with `logsearch`? I thought resource pools were deployment-specific already in which case it seems redundant; but if this sort of thing is good practice I'm interested to learn.